src/main/kotlin/notification/domain/Notification.kt

```diff
@@ -1,5 +1,6 @@
 package notification.domain
 
+import deadline.domain.Time
```

Comment:
I will create an issue for extracting the Time class to shared, since it is a class that can be needed by more than one module.